test/integrations/destinations/bqstream/router/data.ts

```diff
@@ -225,6 +705,157 @@ export const data = [
                 Name: 'bqstream test',
               },
             },
+            {
+              batched: false,
+              destination: {
+                Config: {
+                  datasetId: 'gc_dataset',
+                  eventDelivery: true,
+                  eventDeliveryTS: 1636965406397,
+                  insertId: 'productId',
+                  projectId: 'gc-project-id',
+                  rudderAccountId: '1z8LpaSAuFR9TPWL6fECZfjmRa-',
+                  tableId: 'gc_table',
+                },
+                Enabled: true,
+                ID: '1WXjIHpu7ETXgjfiGPW3kCUgZFR',
+                Name: 'bqstream test',
+              },
+              error: 'Message Type not supported: identify',
+              metadata: [
+                {
+                  jobId: 3,
+                  userId: 'user12345',
+                },
+              ],
+              statTags: {
+                destType: 'BQSTREAM',
+                errorCategory: 'dataValidation',
+                errorType: 'instrumentation',
+                feature: 'router',
+                implementation: 'native',
+                module: 'destination',
+              },
+              statusCode: 400,
+            },
+            {
+              batched: true,
+              batchedRequest: {
+                datasetId: 'gc_dataset',
+                projectId: 'gc-project-id',
+                properties: [
+                  {
+                    count: 20,
+                    insertId: '20',
+                    productId: 20,
+                    productName: 'Product-20',
+                  },
+                  {
+                    count: 20,
+                    insertId: '20',
+                    productId: 20,
+                    productName: 'Product-20',
+                  },
+
+                ],
+                tableId: 'gc_table',
+              },
+              destination: {
+                Config: {
+                  datasetId: 'gc_dataset',
+                  eventDelivery: true,
+                  eventDeliveryTS: 1636965406397,
+                  insertId: 'productId',
+                  projectId: 'gc-project-id',
+                  rudderAccountId: '1z8LpaSAuFR9TPWL6fECZfjmRa-',
+                  tableId: 'gc_table',
+                },
+                Enabled: true,
+                ID: '1WXjIHpu7ETXgjfiGPW3kCUgZFR',
+                Name: 'bqstream test',
+              },
+              metadata: [
+                {
+                  jobId: 5,
+                  userId: 'user123',
+                },
+                {
+                  jobId: 6,
```

Comment:
jobId: 6 is repeated. Can you make sure jobIds are provided correctly.

Also, can we trim down the test case with not needed fields. Very difficult to read.

Reply:
done